node/popn.go

```diff
@@ -998,6 +1050,34 @@ func (nd *node) Add(ctx context.Context, dag ipldformat.DAGService, buf io.Reade
 	return n.Cid(), nil
 }
 
+// selectDAGParams returns the best chunk params according to the file's type
+func selectDAGParams(filepath string, buf io.ReadSeeker) []DAGOption {
+	fileType := utils.DetectFileType(filepath, buf)
+	chunker := DefaultChunker(buf)
+	layout := DefaultLayout
+
+	switch fileType {
+	case utils.FTAudio, utils.FTVideo:
+		chunkSize := int64(units.MiB)
+		chunker = chunk.NewSizeSplitter(buf, chunkSize)
+		layout = trickle.Layout
+
+	case utils.FTImage, utils.FTArchive:
+		chunkSize := int64(units.MiB)
```

Comment:
Do we really need a package for that? Could just do:
```suggestion
		chunkSize := 1 << 20
```

Reply:
imo it was less confusing to write units.MiB, plus it's a const var so not expensive, and it's from a package we are already using

Reply:
Alright that sounds reasonable